cypress/e2e/registration_validation.spec.ts

```diff
@@ -305,9 +305,7 @@ describe('User opens registration form and can see validation errors', () => {
     cy.get(`[data-testid=${dataTestId.registrationWizard.contributors.addContributorButton}]`).first().click();
     cy.get('[data-testid=contributor-modal]').should('be.visible');
     cy.get(`[data-testid=${dataTestId.registrationWizard.contributors.searchField}] input`).type('test');
-    cy.get(`[data-testid=${dataTestId.registrationWizard.contributors.selectEverythingForContributor}]`)
```

Comment:
verdien for `selectEverythingForContributor` kan fjernes fra `dataTestId` også, siden knappen nå er vekke